draft-ietf-gnap-core-protocol.md

```diff
@@ -369,6 +374,11 @@ that returns from the interaction.
     |        |                                  |        |
     |        |<-(9)----- Grant Access ----------|        |
     |        |                                  |        |
+    |        |                                  |        |     +--------+
+    |        |--(10)-- Access API ---------------------------->|   RS   |
```

Comment:
My apologies if it's a bit unrelated to the proposed changes but IMO, a client instance doesn't necessarily access some APIs on an RS. It makes requests to the protected resource. Hence... does `GET /protected.docx HTTP/1.1` fall into `Access API`?

Section 1.2 (Roles) defines a client as

> an application operated by an end-user that consumes resources from one or several RSs

Reply:
The protected resource is an API hosted by the resource server, RS. All of this is inline with each other, but the precision of the definitions can of course be refined!